library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1303,6 +1314,35 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       this.output.format(format);
     }
 
+    private static Pair<List<byte[]>, Long> parseFourCCPrivate(ParsableByteArray buffer)
+            throws ParserException {
+        try {
+            buffer.setPosition(buffer.getPosition()+16); // size, skip width(4), height(4),
+                                                         // planes(2), and bitcount(2)
+            long biCompression = buffer.readLittleEndianUnsignedInt();
+            buffer.setPosition(buffer.getPosition()+20); // skip sizeimage(4), xpel/meter(4),
+                                                       // ypel/meter(4), clrUsed(4), clrImportant(4)
+            List<byte[]> privateData = new ArrayList<>();
+            final int baseBytesLeft = buffer.bytesLeft();
+            byte[] scratchBytes = new byte[baseBytesLeft];
+            buffer.readBytes(scratchBytes, 0, baseBytesLeft);
+
+            int realStartPos;
+            for(realStartPos = 0; realStartPos < (baseBytesLeft - 4); ++realStartPos) {
```

Comment:
Where are the bytes beyond biClrImportant defined (do you have a link to some documentation you can provide, since the documentation I found seems to suggest that biClrImportant is the last thing that should be present).

Also, why is it necessary to search for 0x00,0x00,0x01,0x0f? It appears that on the piece of test media this ends up skipping one byte, which seems a little odd.